### ISSUE TITLE
Max's WTC 2020 Talk

### ISSUE THREAD

**Issue by maxitg** [maxitg/SetReplace#426]
### The problem
I'm giving a talk at the Wolfram Technology Conference 2020. The time is Wed 10/7 at 3 PM CDT. The access is private, so not everybody will be able to see it immediately, but the plan is to upload it to the SetReplace's YouTube channel after all (if there are no copyright issues with doing so).

### Possible solution
I need to prepare the slides and decide what to talk about. The rough plan is:

1. Introduce set substitution systems.
2. Introduce Wolfram models (as a special case of the set substitution systems).
3. Show the animation of the evolution of these, ideally with animated spring-electrical embedding (this will be done simultaneously with the previous two).
4. Do a demo of various SetReplace features.
5. Actionable ask: invite people to come to the SetReplace Q&A on discord (tentatively on Sat 10/10 immediately after the conference). I will send a tweet about that as well. We need to come up with specific things people can work on to discuss during that Q&A.

**Comment by maxitg** [comment]
@aokellermann, @taliesinb, here is the draft of the talk. Please take a look if you have a chance before 3 PM CDT.
Keynote file: http://ipfs.io/ipfs/QmeSDCaUzCGQKtf1bKXHCL48UctVZZ6wL9wLYRUDrf6z1t
Video: http://ipfs.io/ipfs/QmXnJQcqwfDSVsa8zWavu4sxb1h1qPa91s54e4V52GpyjV
Demo file: [SetReplace-Demo-01.nb.zip](https://github.com/maxitg/SetReplace/files/5339988/SetReplace-Demo-01.nb.zip)

The ending needs a bit more work (especially the last slide). I also don't know yet how much time the talk is going to take.

**Comment by aokellermann** [comment]
@maxitg I think the presentation (video) is great. Here are a few suggestions:
* I think there's a bit too much jiggling in the animations.
* Talking about adding a `Module` to the outputs of a rule might be confusing to non-WL people. So maybe in the talk just mention briefly what a `Module` does.
* "Replaces subsets to subsets" is confusing. Perhaps "replaces subsets with newly created subsets"?
* There is nothing in the presentation about what a hypergraph is. It's not a very common concept, so it might be good to define it.
* The penultimate slide is "Demo 2", but there is no demo 2.
* Last slide
  * No link to SetReplace repo
  * No mention of the programming languages used in the project
  * No mention that codebase is open source. People might think that you're looking for new Wolfram employees, since everything else Wolfram does is closed source.

**Comment by taliesinb** [comment]
The video is amazing. Honestly I don't think there is much to improve about it.

**Comment by maxitg** [comment]
Thanks, both of you for the feedback!

I addressed some of the @aokellermann's comments:

- [ ] I actually kind of like the jiggling, I think it makes it more "fun". I could try to experiment with the friction coefficient, but the videos take a long time to generate, so I don't have enough time to do it before the talk, unfortunately.
- [ ] This is a Wolfram Technology Conference, I assume everyone there knows Wolfram Language. If we later adapt this video for the general public, we probably should not be using `Module` at all to explain it.
- [ ] "Replaces subsets to subsets" is confusing. Perhaps "replaces subsets with newly created subsets": fixed.
- [ ] I wasn't going to say anything about hypergraphs, but I probably should. I added a label to the slide where I introduce `WolframModelPlot`.
- [ ] There are two sections in the notebook. The second one is "Demo 2". It's about event ordering and selection functions.
- [ ] I completely redid the last slide. Now it's a lot more similar in style to the rest of the presentation:
<img width="1058" alt="Screen Shot 2020-10-07 at 12 55 53" src="https://user-images.githubusercontent.com/1479325/95368994-6fe2ff80-089c-11eb-9d47-901ff43bbcdd.png">
